packages/sdk/src/react/ui/modals/_internal/components/quantityInput/index.tsx

```diff
@@ -160,10 +95,11 @@ export default function QuantityInput({
 					</Box>
 				}
 				numeric={true}
-				value={$quantity.get()}
+				value={quantity}
 				onChange={(e) => handleChangeQuantity(e.target.value)}
 				width={'full'}
 			/>
+			<Text color="negative">{invalidQuantity ? 'Invalid quantity' : ' '}</Text>
```

Comment:
![fontsize](https://github.com/user-attachments/assets/61798bec-fc10-4b63-965a-efe72328c864)

Can we make their fontsizes match?

Reply:
Thanks, done